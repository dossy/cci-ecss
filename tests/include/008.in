@include "file/not/found.inc";
