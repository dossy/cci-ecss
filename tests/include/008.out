ecss: tests/include/008.in:1:0: Unable to read included stylesheet "file/not/found.inc".
